website/markdown/docs/examples/app-clips.mdx

```diff
@@ -0,0 +1,47 @@
+---
+name: App Clips
+excerpt: 'This page documents how to declare and use App Clip targets.'
+---
+
+import Message from '../components/message'
+
+# App Clips
+
+## Product Types
+
+An App Clip is a small part of your app that’s discoverable at the moment it’s needed. App Clips are fast and lightweight so a user can open them quickly.
+
+For example, this is how an AppClip can be declared:
+
+```swift
+let project = Project(name: "App",
+                      targets: [
+                        Target(name: "App",
+                               platform: .iOS,
+                               product: .app,
+                               bundleId: "io.tuist.App",
+                               infoPlist: "App/Configs/Info.plist",
+                               sources: ["App/Sources/**"],
+                               dependencies: [
+                                    .target(name: "AppClip"),
+                                ]),
+                        Target(name: "AppClip",
+                               platform: .iOS,
+                               product: .appClip,
+                               bundleId: "io.tuist.App.Clip",
```

Comment:
Is this conventional?

Reply:
only requirement is the bundleId needs to be prefixed with parent app bundleId

Reply:
In that case, what about adding a linting check that fails the generation if they are not following the expected convention. We can do the following:
1. Get the app the clip depends on.
2. Get the bundle id of the app and the bundle id of the app clip.
3. If they don't follow the convention, return a linting issue.

Reply:
covered here [here](https://github.com/tuist/tuist/pull/1854/files#r512524873)